.circleci/config.yml

```diff
@@ -160,8 +160,11 @@ jobs:
               export STORE_LOGS_IN_ANY_CASES=true
             fi
             export STORE_JAEGER_TRACES=true
-            pushd test
-            go get github.com/networkservicemesh/cloudtest
+            git clone https://github.com/networkservicemesh/cloudtest.git
```

Comment:
TODO: revert these changes when PR https://github.com/networkservicemesh/cloudtest/pull/3 will be merged

Reply:
Nice idea  :-)